pkgs/shells/zsh/zsh-abbr/default.nix

```diff
@@ -17,10 +17,15 @@ stdenv.mkDerivation rec {
   strictDeps = true;
 
   installPhase = ''
-    install -D zsh-abbr.zsh $out/share/zsh/${pname}/abbr.plugin.zsh
-    # Needed so that `man` can find the manpage, since it looks via PATH
-    mkdir -p $out/bin
+    runHook preInstall
+
+    install zsh-abbr.plugin.zsh zsh-abbr.zsh -Dt $out/share/zsh/${pname}/
+    install completions/_abbr -Dt $out/share/zsh/${pname}/completions/
+
+    # Required for `man` to find the manpage of abbr, since it looks via PATH
     mv man $out/share/man
```

Comment:
Why not use installManPage?

Reply:
Because I wasn't the one that wrote that line and I was not aware of its existence. Going to replace it with that alternative now! Thanks :)